dotcom-rendering/playwright/lib/locators.ts

```diff
@@ -0,0 +1,63 @@
+import type { Locator, Page } from '@playwright/test';
+import { expect } from '@playwright/test';
+
```

Comment:
Convenience methods to assert a selector exists, is visible and their inverse (does not exist and is not visible).